kola/tests/docker/torcx_manifest_pkgs.go

```diff
@@ -39,6 +39,10 @@ func init() {
 		// This test is normally not related to the cloud environment
 		Platforms: []string{"qemu", "qemu-unpriv"},
 		SkipFunc: func(version semver.Version, channel, arch, platform string) bool {
+			// Releases after 3745 don't ship torcx at all (these use sysext)
+			if (0 < version.Compare(semver.Version{Major: 3745})) {
```

Comment:
I think instead of a skipfunc we can use `EndVersion: semver.Version{Major: 3745},` here. If you run the test directly by name this gets ignored but normally it's evaluated.

Reply:
Agreed, will change. Coincidentally this was the first file I touched when starting to work on sysext testing; didn't know at the time that we had this cool `MinVersion` / `EndVersion` mechanism.

Reply:
Done - resolving.